misc/python/materialize/parallel_workload/action.py

```diff
@@ -1064,6 +1064,12 @@ def __init__(
             "16",
             "1000",
         ]
+        self.flags_with_values["persist_compaction_memory_bound_bytes"] = [
```

Comment:
Above we limit the size of blobs to 1 MiB, 16 MiB, and 128 MiB. Should those limits be reflected here too? Assuming that we're tuning these to fit a very limited number of Parts/blobs in-memory at a given time during compaction.